README.md

```diff
@@ -16,39 +16,58 @@
 
 ```bash npm2yarn
 npm i puppeteer # Downloads compatible Chrome during installation.
-npm i puppeteer-core # Alternatively, install as a library, without downloading Chrome.
+npm i puppeteer-core # Alternatively, install as a library, without downloading browsers.
 ```
 
 ## Example
 
 ```ts
 import puppeteer from 'puppeteer';
-// Or import puppeteer from 'puppeteer-core';
 
-// Launch the browser and open a new blank page
-const browser = await puppeteer.launch();
-const page = await browser.newPage();
+// Launch Chrome.
+const chrome = await puppeteer.launch({
+  browser: 'chrome',
+});
 
-// Navigate the page to a URL.
-await page.goto('https://developer.chrome.com/');
+// Launch Firefox.
+const firefox = await puppeteer.launch({
+  browser: 'firefox',
+});
```

Comment:
I wonder if we should make both use BiDi...? Can't decide what's the best message to send: you can do all the things equally in Firefox and Chrome using CDP and BiDi *or* look you can do the same things with the same protocol in both browsers.

I am leaning towards the later.